mm2src/coins/eth/v2_activation.rs

```diff
@@ -238,20 +268,19 @@ pub async fn eth_coin_from_conf_and_request_v2(
         })
     }
 
-    if web3_instances.is_empty() {
+    if !is_metamask && web3_instances.is_empty() {
```

Comment:
Related to previous comment: I think it might be worth trying to avoid these `if is_metamask` conditions. Though, I don't really insist on it, and it might be also done during next iteration(s).